src/main/java/mod/surviving_the_aftermath/capability/RaidData.java

```diff
@@ -42,6 +46,7 @@ public boolean enterPortal(Entity entity) {
 		if (level.structureManager().getAllStructuresAt(pos).containsKey(
 				level.registryAccess().registryOrThrow(Registries.STRUCTURE).get(ModStructures.NETHER_RAID))) {
 			if (entity instanceof Player && noRaidAt(pos)) {
+				entity.sendSystemMessage(Component.nullToEmpty(I18n.get("raid.surviving_the_aftermath.begin")));
```

Comment:
Just use Component.translatable("raid.surviving_the_aftermath.begin") instead of Component.nullToEmpty(I18n.get("raid.surviving_the_aftermath.begin"))